tests/integration_tests/functional/test_cmd_line_start.py

```diff
@@ -206,6 +225,73 @@ def test_config_with_default_limit(test_microvm_with_api, vm_config_file):
     assert stdout.encode("utf-8") == response_err.encode("utf-8")
 
 
+@pytest.mark.parametrize(
+    "vm_config_file",
+    ["framework/vm_config_with_mmdsv1.json"]
+)
+def test_config_start_with_mmds_default(test_microvm_with_api, vm_config_file):
```

Comment:
I would try to rename this to make it more consistent with `test_with_config_and_metadata_no_api`.

Reply:
well this one actually has an api. should we add tests for both scenarios? checking config start with mmds default both with and without an api